packages/website/src/styles/index.js

```diff
@@ -0,0 +1,98 @@
+import styled from 'styled-components';
+
+const logo_height = "calc(var(--size-base)*2/1.75)";
+
+const Index = styled.div`
+  h1 {
```

Comment:
This is not an adequate way to declaring styles. We want to favor `styled-components` for each tag and avoid leaking styles around.

Reply:
And this too